lib/ecto_ldap/adapter.ex

```diff
@@ -9,6 +10,15 @@ defmodule Ecto.Ldap.Adapter do
     IO.inspect(preprocess)
     IO.inspect(options)
 
+    {:ok, connection} = Exldap.connect
+    {:ok, search_results} = Exldap.search_field(
+        connection,
+        "ou=users,dc=puppetlabs,dc=com",
+        "mail",
+        'jeff.weiss@puppetlabs.com')
```

Comment:
Right now I think I'd prefer to not rely on `exldap`. I'd rather first get us what we need just using straight `eldap`.